backend/schemas/user.py

```
@@ -1,8 +1,62 @@
+# ✅ Signup Schema (Minimal Fields)
 class UserCreate(UserBase):
     password: str
+
+# ✅ Profile Completion Schema (Handled Separately)
+class ProfileCompletionRequest(BaseModel):
+    university_name: str
+    department: str
+    fields_of_interest: List[str]  # Expecting a list
+
+    @classmethod
+    def as_form(
```

Ruff (0.8.2)

Missing return type annotation for classmethod `as_form`

(ANN206)